debian/autobake-deb.sh

```diff
@@ -98,6 +98,13 @@ then
   sed 's/libcurl4/libcurl3/g' -i debian/control
 fi
 
+# From Debian Bullseye/Ubuntu Groovy, liburing replaces libaio
+if ! apt-cache madison liburing-dev | grep 'liburing-dev' >/dev/null 2>&1
+then
+  sed 's/liburing-dev/libaio-dev/g' -i debian/control
+  sed '/-DIGNORE_AIO_CHECK=YES/d' -i debian/rules
```

Comment:
Should we remove more matches, such as `-DIGNORE_AIO_CHECK:BOOL=ON`?
I think that we would want to add `-DWITH_URING=ON` to `debian/rules` if the library is available. (This is waiting for MDEV-24883 to actually implement the build option.)

Reply:
No, it only needs to match the exact lines in `debian/*` that exist there right now. It is not meant to be able to cope from arbitrary manual changes.